src/themes/material/elements/icon.overrides

```diff
@@ -1524,16 +1524,16 @@ i.icon.lock.outline::before {
 i.icon.looks::before {
     content: "\e3fc";
 }
-i.icon.looks.3::before {
+i.icon.looks.three::before {
```

Comment:
class name starting with digit is not good, is this change ok? what are the source data? it seems some numbers are missing, especially the last change looks fishy

Reply:
It's ok, we did that for the default font awesome icons as well

Reply:
The official way to make it work anyway would be to escape it 
`.\32 `
Or use the attribute selector
`[class*="2"]`
But (!) The ever since existing notation `.2` in the material theme, never worked anyway, so nobody was ever complaining about, so most probably nobody ever used those icons 😉

So, again, changing them to words is totally ok and consistent to the default

Reply:
A lot of changes have been done manually, please double check them, if all good, PR is finished.